addons/point_of_sale/models/pos_config.py

```diff
@@ -634,7 +634,9 @@ def open_ui(self):
             raise UserError(_("You do not have permission to open a POS session. Please try opening a session with a different user"))
 
         if not self.current_session_id:
-            self._check_before_creating_new_session()
+            res = self._check_before_creating_new_session()
```

Comment:
`_check_before_creating_new_session` returns nothing 🤔

Reply:
Ah I didn't see the override in pos_blackbox_be but you should add the return in all override in all modules